romana-install/roles/provision/create-aws-hosts/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Generate CloudFormation Template
+  template: src="aws_cf_small_cluster.json" dest="{{ stack_data_dir }}/cloudformation.json"
+  register: cf
+
+- name: Create stack with Cloudformation. This may take around 5 minutes.
```

Comment:
Maybe we should call this "create deployment nodes" or something, since "create stack" sounds too close to Devstack or OpenStack and that's not exactly what we are doing here.